src/components/Form/Toggle.tsx

```diff
@@ -5,14 +5,12 @@ import React, {
   ComponentClass,
   FunctionComponent,
 } from 'react';
-import { HtmlLabel, HtmlInput, HtmlSpan } from '../../reset';
 import { idGenerator } from '../../utils/uuid';
-import classnames from 'classnames';
+import { ToggleInput } from './ToggleInput';
+import { ToggleButton } from './ToggleButton';
 
-const baseClassName = 'fi-toggle';
-const toggleDisabledClassName = `${baseClassName}--disabled`;
-const toggleInputClassName = `${baseClassName}_input`;
-const toggleLabelClassName = `${baseClassName}_label`;
+export { ToggleButton, ToggleInput };
```

Comment:
Does these need to be exported also separately?

Reply:
No longer exporting the ToggleButton.
This should now be solved with recent commit.